src/main/resources/db/changelog/changes/011-create-product-bundle.sql

```diff
@@ -1,6 +1,6 @@
-CREATE TABLE IF NOT EXISTS sets (
+CREATE TABLE IF NOT EXISTS products_bundle (
```

Comment:
You can't just change migrations. They are supposed to be immutable. If you want to delete table `sets` you just create new migration  that deletes this table. After that you create another migration that creates table `products_bundle`